source/bin/tvm.rs

```diff
@@ -154,10 +166,8 @@ mod tests {
     #[test]
     fn case_usage() {
         assert_eq!(
-            "Usage: tvm [options] [file]\n".to_string() +
-            "Options:\n" +
-            "    -v, --version    Print version.\n" +
-            "    -h, --help       This help.",
+            "Usage: tvm [options] [file]\nOptions:\n    -v, --version    \
```

Comment:
This is a very basic unit test here ;-). There is not really something to test :-p.